src/test/java/com/znsio/teswiz/services/UnirestService.java

```diff
@@ -0,0 +1,44 @@
+package com.znsio.teswiz.services;
+
+import kong.unirest.HttpResponse;
+import kong.unirest.JsonNode;
+import kong.unirest.Unirest;
+import org.apache.log4j.Logger;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class UnirestService {
+
+    private static final Logger LOGGER = Logger.getLogger(UnirestService.class);
+    private static final Unirest unirest = new Unirest();
+
+    private static Unirest getUnirestObj(){
+        unirest.config().verifySsl(false).socketTimeout(1000).connectTimeout(500);
```

Comment:
why timeout of 1sec and 500ms?

Reply:
This was wrongly committed as we were trying out something with configurations. later on we inferred that it is not required so removed it.